troposphere/static/js/AppRoutes.react.js

```diff
@@ -66,22 +67,22 @@ define(function (require) {
       </Route>
 
       <Route name="providers" handler={ProvidersMaster}>
-        <Route name="provider" path=":providerId" handler={ProviderDetailsPage}/>
-        <DefaultRoute handler={ProviderDetailsPage}/>
+        <DefaultRoute handler={ProviderListSection} />
+        <Route name="provider" path=":id" handler={ProviderDetail}/>
+        <Route name="all-providers" path="/" handler={ProviderListSection} />
       </Route>
 
       <Route name="help" handler={HelpPage}/>
       <Route name="settings" handler={SettingsPage}/>
 
       <Route name="admin" handler={AdminMaster}>
-        <Route name="identity-membership-manager" path="users" handler={IdentityMembershipMaster}/>
+        <DefaultRoute name="identity-membership-manager" path="users" handler={IdentityMembershipMaster}/>
```

Comment:
This is not correct. `DefaultRoute` does **not** have a `path` property. If you look at the warnings in the DevTools console, it'll state that 

```
_DefaultRoute doesn't have a path property_
```

We just need to include the change from this commit: 538b325b8e0e340b911060769dfc4da26dc8991e

In short, it would be: 

```html
      <Route name="admin" handler={AdminMaster}>
        <Route name="identity-membership-manager" path="users" handler={IdentityMembershipMaster}/>
        .... 
        <DefaultRoute handler={IdentityMembershipMaster}/>
      </Route>
```